print-radius.c

```diff
@@ -541,6 +609,68 @@ print_attr_string(netdissect_options *ndo,
            data++;
            length--;
         break;
+      case EAP_MESSAGE:
+           type = *data;
```

Comment:
Tcpdump master branch has switched to `EXTRACT_U_1()` and `EXTRACT_S_1()` for single-byte fetching; multiple-byte fetching macros have been renamed respectively (amount of bytes instead of 1 above). Please update the code to use those new macros.